app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,42 +33,103 @@ export default function NoteTable({
                 }}
               />
             </th>
-            <th className="text-left">Title</th>
-            <th className="text-left">Description</th>
+            <th className="text-left w-1/6 text-grey-tableTitle text-tableTitle">
+              Title
+            </th>
+            <th className="text-left w-1/5 text-grey-tableTitle text-tableTitle">
+              Description
+            </th>
+            <th className="text-center w:auto text-grey-tableTitle text-tableTitle">
+              Tags
+            </th>
+            <th className="text-center w:auto text-grey-tableTitle text-tableTitle">
+              Created Date
+            </th>
+            <th className="text-center w:auto text-grey-tableTitle text-tableTitle">
+              Due Date
+            </th>
+            <th className="text-center w:auto text-grey-tableTitle text-tableTitle">
+              Contact
+            </th>
+            <th className="text-center w-1/12 text-grey-tableTitle text-tableTitle"></th>
           </tr>
         </thead>
         <tbody>
-          {notes.map(note => (
-            <tr
-              key={note.id}
-              className={"cursor-pointer bg-white hover:bg-gray-50"}
-            >
-              <td>
-                <Checkbox
-                  checked={selectedNoteIds.includes(note.id)}
-                  onClick={event => {
-                    event.stopPropagation();
-                    const index = selectedNoteIds.indexOf(note.id);
-
-                    if (index > -1) {
-                      setSelectedNoteIds([
-                        ...selectedNoteIds.slice(0, index),
-                        ...selectedNoteIds.slice(index + 1),
-                      ]);
-                    } else {
-                      setSelectedNoteIds([...selectedNoteIds, note.id]);
-                    }
-                  }}
-                />
-              </td>
-              <td>
-                <div className="flex flex-row items-center justify-start text-gray-900">
-                  {note.title}
-                </div>
-              </td>
-              <td>{note.description}</td>
-            </tr>
-          ))}
+          {notes.map(note => {
+            const contact = {
+              name: "Oliver Smith",
+            };
+            const createdDate = format(new Date(note.created_at), "MMM, d y");
+            const dueDate = note?.due_date
+              ? format(new Date(note.due_date), "MMM, d y")
+              : "--";
+            const badgeColor =
+              issueTypeBadgeColors[notes.issueType || "internal"];
+            return (
+              <tr
+                key={note.id}
+                className={"group cursor-pointer bg-white hover:bg-gray-50"}
```

Comment:
Please remove this since we are adding `nui-table--hover` class for hover effects.

Reply:
Resolved in #9